src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/MandateForChangeTest.kt

```diff
@@ -27,11 +15,8 @@ class MandateForChangeTest : MockedEndpointsTestBase() {
     setupRestrictiveRequirements(crn)
     setupMaleOffenderWithRegistrations(crn)
 
-    listener.listen(calculationMessage(crn))
-    val tier = repo.findFirstByCrnOrderByCreatedDesc(crn)
-
-    Assertions.assertThat(tier?.data?.change?.tier).isEqualTo(ChangeLevel.ZERO)
-    Assertions.assertThat(tier?.data?.protect?.tier).isEqualTo(ProtectLevel.A)
+    calculateTierFor(crn)
+    expectTierCalculation("A0")
```

Comment:
This looks much nicer, I'm glad we don't have a requirement to display them separately.